content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -194,19 +194,40 @@ authentication is needed.
 On the first usage of a GDrive [remote](/doc/command-reference/remote), for
 example when trying to `dvc push` for the first time after adding the remote
 with a [valid URL](#url-format), DVC will prompt you to visit a special Google
-authorization web page. There you'll need to sign into your Google account. The
+authentication web page. There you'll need to sign into your Google account. The
 [auth process](https://developers.google.com/drive/api/v2/about-auth) will ask
 you to grant DVC the necessary permissions, and produce a verification code
 needed for DVC to complete the connection. On success, the necessary credentials
 will be saved in a Git-ignored file, located in
-`.dvc/tmp/gdrive-user-credentials.json`.
+`.dvc/tmp/gdrive-user-credentials.json` and they will be used automatically next
+time you run DVC.
 
 ⚠️ In order to prevent unauthorized access to your Google Drive, **do not share
 these credentials with others**. Each team member should go through this process
 individually.
 
+If you use multiple GDrive remotes, by default they will be sharing the same
+`.dvc/tmp/gdrive-user-credentials.json` credentials file. It can be overridden
```

Comment:
Maybe we don't need to repeat the file name since it's mentioned in in the previous paragraph (before the warning).